### DISCUSSION TITLE
NFS with user and pass

### DISCUSSION THREAD

**Discussion by frit0-rb** [longhorn/longhorn#7198] [Q&A]
Good morning,

We are wondering if it possible to add a NFS Backup target with user and password, is that possible?

Thanks

**Comment by derekbit** [comment]
Currently, no.

Why not use object storage such s3 or s3-compatible solutions such as MinIO?

**Reply by frit0-rb** [reply]
Because, in our case, the Minio will be in the same cluster and if we lost the cluster we cant recover the backups too.
